addons/sys_core/stringtable.xml

```diff
@@ -405,6 +405,9 @@
             <Portuguese>Detecção de Voz de IA</Portuguese>
             <Italian>Rilevamento Voce IA</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_core_rememberUsedRackRadios_displayName">
+            <English>Remember Vehicle Racks In Use</English>
```

Comment:
Agree. @tbeswick96 Would be cool if you find the time to change this. 
Needed to get in and out of a vehicle a lot in our mission yesterday and this would really help to keep the workload down.